terraform/modules/pmap-service/variables.tf

```diff
@@ -86,3 +86,9 @@ variable "pmap_specific_envvars" {
     EOT
   default     = {}
 }
+
+variable "enable_dead_lettering" {
+  description = "Enables the creation of a dead letter topic for the pub/sub push subscription."
+  type        = bool
+  default     = true
```

Comment:
Do we want the default to be true? for example for ci, we probably don't need it since the dlq will be deleted every ci run. either change the default or we can update the ci to set it to false?